nshankaz12/SBPD-Help-Desk
# Implemented "Resolve a ticket"

Implemented "Resolve a ticket". 

To test it,
1. Go to a certain ticket you created.
2. Click on "Close Ticket" button.
3. Test out cancel button to see if pop up goes away.
4. Insert a resolution in the text box and click close ticket.
5. If you then go to My Tickets. That ticket should be marked as closed. 
6. Or you can go into our DB and see if the change got reflected on that ticket.
